### PR TITLE
add section 1 content.zh-CN.md

### PR DESCRIPTION
**Pull request by thrillerist**

### CODE REVIEW

File: components/sections/preface/content.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+我们通常用线条来绘制2D图形，大致分为两种线条：直线和曲线。不论我们动手还是用电脑，都能很容易地画出第一种线条。只要给电脑起点和终点，砰！直线就画出来了。没什么好疑问的。
+
+然而，绘制曲线却是个大问题。虽然我们可以很容易地徒手画出曲线，但除非给出描述曲线的数学函数，不然计算机无法画出曲线。实际上，画直线时也需要数学函数，但画直线所需的方程式很简单，我们在这里不去考虑。在计算机看来，所有线条都是“函数”，不管它们是直线还是曲线。然而，这就表示我们需要找到能在计算机上表现良好的曲线方程。这样的曲线有很多种，在本文我们主要关注一类特殊的、备受关注的函数，基本上任何画曲线的地方都会用到它：贝塞尔曲线。
+
+它们是以[Pierre Bézier](https://en.wikipedia.org/wiki/Pierre_B%C3%A9zier)命名的，尽管他并不是第一个，或者说唯一“发明”了这种曲线的人，但他让世界知道了这种曲线十分适合设计工作（在1962年为Renault工作并发表了他的研究）。有人也许会说数学家[Paul de Casteljau](https://en.wikipedia.org/wiki/Paul_de_Casteljau)是第一个发现这类曲线特性的人，在Citroën工作时，他提出了一种很优雅的方法来画这些曲线。然而，de Casteljau没有发表他的工作，这使得“谁先发现”这一问题很难有一个确切的答案。
+贝塞尔曲线本质上是伯恩斯坦多项式，这是[Sergei Natanovich Bernstein](https://en.wikipedia.org/wiki/Sergei_Natanovich_Bernstein)研究的一种数学函数，关于它们的出版物至少可以追溯到1912年。无论如何，这些都只是一些冷知识，你可能更在意的是这些曲线很方便：你可以连接多条贝塞尔曲线，并且连接起来的曲线看起来就像是一条曲线。甚至，在你在Photoshop中画“路径”或使用一些像Flash、Illustrator和Inkscape这样的矢量绘图程序时，所画的曲线都是贝塞尔曲线。
+
+那么，要是你自己想编程实现它们呢？有哪些陷阱？你怎么画它们？包围盒是怎么样的，怎么确定交点，怎么拉伸曲线，简单来说：你怎么对曲线做一切你想做的事？这就是这篇文章想说的。准备好学习一些数学吧!
+
+—Pomax ( 推特账号, [@TheRealPomax](https://twitter.com/TheRealPomax) )
```
Comment (Pomax) [inline components/sections/preface/content.zh-CN.md]: should there be spaces between `( 推特账号` and between `@TheRealPomax )`?

Comment (thrillerist) [inline components/sections/preface/content.zh-CN.md]: It shows incorrectly without these space when I edit the file in github editor. But you can remove them if they're not necessary. ( and remove other unnecessary spaces )

Comment (Pomax) [inline components/sections/preface/content.zh-CN.md]: Yeah the github preview is probably not the best preview, it doesn't quite have all the conversion rules that kick in when the files are processed for the project. I'll merge this in, thanks!

Comment (thrillerist) [inline components/sections/preface/content.zh-CN.md]: If you build the chinese version online page, please send me the link. :)

Comment (Pomax) [inline components/sections/preface/content.zh-CN.md]: I'll update the localization issue when there is a deployed (partial) locale to check out. I'll probably have some time this evening to get that deployed. Thanks for helping out!

**Pull request merged**